.eslintignore

```diff
@@ -1,20 +1,35 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Previously .gitignore was the file for ignoring lint. Modified it to use .eslintignore instead. Hence adding some ignore files that was in .gitignore, in this file.